app/controllers/UserController.php

```diff
@@ -166,7 +166,8 @@ public function sendFeedback () {
       $feedbackData = [
         'user' => $params->user,
         'code' => $params->code,
-        'feedback' => $params->feedback
+        'feedback' => $params->feedback,
+        'created_at' => date('Y-m-d H:i:s', time())
```

Comment:
Works for me ( see below ). Remember to run `php artisan migrate` to create the `feedback` table. :) 

<img width="547" alt="screen shot 2016-12-16 at 11 22 41" src="https://cloud.githubusercontent.com/assets/4658706/21259567/53b0ac6c-c382-11e6-99f2-2452b830a478.png">